src/Root.tsx

```diff
@@ -3,14 +3,22 @@ import { observer } from "mobx-react-lite";
 import { DiagramCanvas } from "./app/DiagramCanvas";
 import { Box } from "@mui/material";
 import { BottomPanel } from "./app/BottomPanel";
+// import { TestModal, TestModalSymbol } from "./app/TestModal";
```

Comment:
Please remember to remove redundant test codes for Line 6-8, 11-13 and 18

Reply:
I removed some text code in DiagramCanvas.tsx but not the one in Root.tsx

I think they might be useful later when we develop another modal, and it can be an example for us. I am happy to remove it after sprint 1.

Reply:
Okay, thank you. I believe the remaining codes are looks fine to me